dio/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ See the [Migration Guide][] for the complete breaking changes list.**
 - Split the Web implementation to `package:dio_web_adapter`.
 - Add FusedTransformer for improved performance when decoding JSON.
 - Improves `InterceptorState.toString()`.
+- Set FusedTransformer as the default transformer.
```

Comment:
Can you combine this with the previous FusedTransformer entry? This would make it easier to understand.

Reply:
yes, done